datashuttle/datashuttle.py

```diff
@@ -617,36 +580,48 @@ def _make_directory_trees(
         ):
             return
 
+        top_level_dir = self._top_level_dir_name
+
+        for sub in sub_names:
+
+            sub_path = self._join("local", [top_level_dir, sub])
+
+            utils.make_dirs(sub_path)
+
+            self.make_experiment_type_folders(experiment_type, sub_path, "sub")
+
+            for ses in ses_names:
+
+                ses_path = self._join("local", [top_level_dir, sub, ses])
+
+                utils.make_dirs(ses_path)
+
+                self.make_experiment_type_folders(
+                    experiment_type, ses_path, "ses"
+                )
+
```

Comment:
Consider printing a message at the end of this function. Sth like "Directories were successfully created". Currently it prints nothing if it's successfull. Optionally you could output all the paths that were created. This way the user will immediately see if they made a mistake

Reply:
Nice idea, this is now logged and when the function is comleteled it says  ([commit](https://github.com/neuroinformatics-unit/datashuttle/pull/66/commits/a9f45f0f811729af675ee5e44f22fec189c4e9f4))

`         f"Finished making directories. For log of all created "
            f"directories, pleasee see {self._logging_path}"`